client/src/components/ToTopAnchor.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div v-if="showAnchor" class="toTopAnchor" @click="backToHeader">
+    <i class="fas fa-arrow-up"></i>
+  </div>
+</template>
+
+<script>
+
+export default {
+  data () {
+    return {
+      showAnchor: false,
+      scrollLimit: 150
+    }
+  },
+  mounted () {
+    window.addEventListener('scroll', this.onScroll)
```

Comment:
Beware for future projects that events like this must be always cleared upon destroying the component - otherwise you get memory leaks, aka:

```
beforeDestroy() {
    window.removeEventListener....
}
```